contracts/test/mocks/MockGateway.sol

```diff
@@ -15,38 +15,19 @@ contract MockGateway is IAxelarGatewayWithToken {
         External
     }
 
-    /// @dev Removed slots; Should avoid re-using
-    // bytes32 internal constant KEY_ALL_TOKENS_FROZEN = keccak256('all-tokens-frozen');
-    // bytes32 internal constant PREFIX_TOKEN_FROZEN = keccak256('token-frozen');
-
-    /// @dev Storage slot with the address of the current factory. `keccak256('eip1967.proxy.implementation') - 1`.
-    bytes32 internal constant KEY_IMPLEMENTATION =
-        bytes32(0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc);
-
-    // AUDIT: slot names should be prefixed with some standard string
     bytes32 internal constant PREFIX_COMMAND_EXECUTED = keccak256('command-executed');
     bytes32 internal constant PREFIX_TOKEN_ADDRESS = keccak256('token-address');
     bytes32 internal constant PREFIX_TOKEN_TYPE = keccak256('token-type');
     bytes32 internal constant PREFIX_CONTRACT_CALL_APPROVED = keccak256('contract-call-approved');
     bytes32 internal constant PREFIX_CONTRACT_CALL_APPROVED_WITH_MINT = keccak256('contract-call-approved-with-mint');
-    bytes32 internal constant PREFIX_TOKEN_DAILY_MINT_LIMIT = keccak256('token-daily-mint-limit');
     bytes32 internal constant PREFIX_TOKEN_DAILY_MINT_AMOUNT = keccak256('token-daily-mint-amount');
```

Comment:
can be removed as well?

Reply:
yeah, `PREFIX_TOKEN_DAILY_MINT_AMOUNT ` is being used `tokenMintAmount` function and `tokenMintAmount` function is being used for `mintToken`. There are two places referencing `gateway.mintToken` in unit tests.

Reply:
Had a discussion, we don't need to track the daily mint amount. removed non relevant logic.